pages.pt_BR/common/fdp.md

```diff
@@ -0,0 +1,25 @@
+# fdp
+
+> Renderiza uma imagem de um gráfico de rede `force-directed` a partir de um arquivo `graphviz`.
+> Layouts: `dot`, `neato`, `twopi`, `circo`, `fdp`, `sfdp`, `osage` & `patchwork`.
+> Mais informações: <https://graphviz.org/doc/info/command.html>.
+
+- Renderiza uma imagem `png` com um nome de arquivo baseado no nome do arquivo de entrada e formato de saída (-O maiúsculo):
+
+`fdp -T {{png}} -O {{caminho/para/entrada.gv}}`
+
+- Renderiza uma imagem `svg` com o nome do arquivo de saída especificado (-o minúsculo):
+
+`fdp -T {{svg}} -o {{caminho/para/imagem.svg}} {{caminho/para/entrada.gv}}`
+
+- Renderiza a saída nos formatos `ps`, `pdf`, `svg`, `fig`, `png`, `gif`, `jpg`, `json`, ou `dot`:
```

Comment:
The following description fix could be applied after the [original](https://github.com/tldr-pages/tldr/blob/main/pages/common/fdp.md) page is refreshed:

```suggestion
- Renderiza a saída nos formatos:
```